renovate.json5

```diff
@@ -14,5 +14,13 @@
         "org.jetbrains.kotlin:kotlin",
       ],
     },
+    {
+      // ktlint breaks their API constantly and regularly needs a Spotless update to work.
```

Comment:
Supposedly `0.49.0` should be the last set of breaking changes (see "API Changes & RuleSet providers & Reporter Providers" [here](https://github.com/pinterest/ktlint/releases/tag/0.49.0)) 🤞

Reply:
Oh nice. If it goes well we can drop this. It doesn't force them together anyway, only merges when there are both.